[2020-10-20 14:08:43,616] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 302 0
[2020-10-20 14:08:46,845] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1857
[2020-10-20 14:08:46,993] [base.py:93] [base:get_response] [WARNING]- Not Found: /favicon.ico
[2020-10-20 14:08:46,994] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /favicon.ico HTTP/1.1" 404 2191
[2020-10-20 14:09:11,311] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1989
[2020-10-20 14:09:15,104] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2020-10-20 14:09:15,145] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 4683
[2020-10-20 14:09:21,983] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/ HTTP/1.1" 200 3304
[2020-10-20 14:09:22,034] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 14:09:24,551] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 14:09:24,588] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
[2020-10-20 14:09:24,588] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 14:09:24,598] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
[2020-10-20 14:09:24,600] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
[2020-10-20 14:09:24,606] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
[2020-10-20 14:20:09,945] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4706
[2020-10-20 14:20:10,023] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2644
[2020-10-20 14:20:18,696] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw?ran=0.7623445664194008 HTTP/1.1" 200 2401
[2020-10-20 14:20:20,281] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw?ran=0.3545711350662404 HTTP/1.1" 200 2788
[2020-10-20 14:20:21,926] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw?ran=0.9612649008506067 HTTP/1.1" 200 2938
[2020-10-20 14:20:27,895] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 14:20:27,910] [views.py:232] [views:index] [INFO]- 缓存中的key为：4e19d1c8129c11ebb6d15800e3fb0696----浏览器中的key为:4e19d1c8129c11ebb6d15800e3fb0696
[2020-10-20 14:20:27,913] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 5607
[2020-10-20 14:29:27,657] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:4e19d1c8129c11ebb6d15800e3fb0696
[2020-10-20 14:29:27,666] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 14:29:27,675] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 7396
[2020-10-20 14:29:27,714] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2949
[2020-10-20 14:29:46,655] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4706
[2020-10-20 14:29:46,711] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2648
[2020-10-20 14:37:21,458] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 200 4820
[2020-10-20 14:37:21,505] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 3008
[2020-10-20 14:37:30,117] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 14:37:30,132] [views.py:232] [views:index] [INFO]- 缓存中的key为：a5d89868129d11eb96a55800e3fb0696----浏览器中的key为:a5d89868129d11eb96a55800e3fb0696
[2020-10-20 14:37:30,144] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 39079
[2020-10-20 14:41:10,952] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 40871
[2020-10-20 14:41:10,984] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:41:10,989] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:41:10,990] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:41:10,991] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:41:11,356] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:41:11,356] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:41:11,364] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:41:11,369] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:41:11,370] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:41:11,371] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:04,699] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 40187
[2020-10-20 14:42:04,736] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:04,741] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:04,743] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:04,743] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:04,831] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:04,832] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:04,840] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:04,841] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:34,381] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 40187
[2020-10-20 14:42:34,422] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:34,427] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:34,428] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:34,430] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:34,485] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:34,486] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:34,494] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:34,495] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:38,891] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 39767
[2020-10-20 14:42:38,926] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:38,934] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:38,935] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:38,936] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:38,962] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:38,963] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:38,970] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:38,970] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:51,131] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 40187
[2020-10-20 14:42:51,163] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:51,170] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:51,171] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:51,172] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:51,222] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:51,223] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:51,228] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:51,229] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:54,474] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 40187
[2020-10-20 14:42:54,513] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:54,516] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:54,524] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:54,525] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:42:54,555] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:42:54,555] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:42:54,562] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:42:54,563] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:43:18,731] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 39843
[2020-10-20 14:43:18,769] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:43:18,775] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:43:18,776] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:43:18,776] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:43:18,830] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:43:18,831] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:43:18,837] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:43:18,838] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:45:18,713] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 37043
[2020-10-20 14:45:18,750] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:45:18,755] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:45:18,756] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:45:18,759] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:45:18,800] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:45:18,801] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:45:18,809] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:45:18,811] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:45:18,815] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:45:18,816] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:52,529] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:47:52,565] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:52,570] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:52,572] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:52,573] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:52,874] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:52,875] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:52,882] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:52,888] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:52,888] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:52,890] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:54,995] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:47:55,027] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:55,029] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:55,032] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:55,033] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:55,065] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:55,066] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:55,076] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:55,079] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:55,082] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:55,083] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:56,210] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:47:56,242] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:56,247] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:56,248] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:56,248] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:56,276] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:56,282] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:56,282] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:56,283] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:57,492] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:47:57,522] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:57,525] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:57,527] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:57,527] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:57,806] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:57,807] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:57,816] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:57,820] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:57,821] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:57,822] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:59,258] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:47:59,291] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:59,297] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:59,298] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:59,299] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:47:59,329] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:47:59,330] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:47:59,339] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:47:59,339] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:48:01,191] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36698
[2020-10-20 14:48:01,220] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:48:01,224] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:48:01,225] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:48:01,225] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:48:01,263] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:48:01,269] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:48:01,270] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:48:01,270] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:49:07,055] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/test HTTP/1.1" 200 36664
[2020-10-20 14:49:07,096] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:49:07,102] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:49:07,103] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:49:07,103] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:49:07,162] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:49:07,162] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:49:07,172] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:49:07,175] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:49:07,176] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:49:07,176] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:51:06,854] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:a5d89868129d11eb96a55800e3fb0696
[2020-10-20 14:51:06,871] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 14:51:06,879] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4706
[2020-10-20 14:51:06,932] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2457
[2020-10-20 14:51:15,603] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 14:51:15,620] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:51:15,636] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 40270
[2020-10-20 14:52:30,878] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:52:30,881] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42173
[2020-10-20 14:52:30,927] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:52:30,932] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:52:30,933] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:52:30,934] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:52:31,004] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:52:31,005] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:52:31,016] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:52:31,023] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:52:31,023] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:52:31,024] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:54:50,353] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:54:50,368] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42176
[2020-10-20 14:54:50,426] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:54:50,435] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:54:50,436] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:54:50,437] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:54:50,525] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:54:50,526] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:54:50,541] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:54:50,544] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:54:50,546] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:54:50,548] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:55:47,571] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:55:47,585] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41823
[2020-10-20 14:55:47,625] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:47,626] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:47,632] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:55:47,633] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:55:47,685] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:47,686] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:47,696] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:47,697] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:47,705] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:55:47,705] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:55:48,448] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:55:48,451] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41823
[2020-10-20 14:55:48,481] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:48,487] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:55:48,488] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:48,491] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:55:48,530] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:48,531] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:48,541] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:55:48,545] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:55:48,547] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:55:48,548] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:08,785] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:08,800] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:08,838] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:08,839] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:08,850] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:08,853] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:08,895] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:08,895] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:08,903] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:08,905] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:10,557] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:10,560] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:10,598] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:10,604] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:10,606] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:10,607] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:10,655] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:10,656] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:10,664] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:10,664] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:11,471] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:11,473] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:11,505] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:11,506] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:11,511] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:11,512] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:11,548] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:11,548] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:11,558] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:11,558] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:12,193] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:12,196] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:12,229] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:12,237] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:12,238] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:12,238] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:12,278] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:12,279] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:12,288] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:12,289] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:12,853] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:12,857] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:12,893] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:12,897] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:12,898] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:12,898] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:12,939] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:12,940] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:12,947] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:12,953] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:12,954] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:12,955] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:13,541] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:13,544] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:13,576] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:13,583] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:13,584] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:13,586] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:13,624] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:13,625] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:13,634] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:13,634] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:14,093] [views.py:232] [views:index] [INFO]- 缓存中的key为：a0eb879412a011ebbcd65800e3fb0696----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:14,096] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 41734
[2020-10-20 14:56:14,129] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:14,132] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:14,133] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:14,135] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:14,172] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:56:14,175] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:56:14,176] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:56:14,177] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:56:26,984] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:a0eb879412a011ebbcd65800e3fb0696
[2020-10-20 14:56:26,992] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 14:56:26,999] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 14:56:27,043] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2643
[2020-10-20 14:56:32,860] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw?ran=0.816244472783106 HTTP/1.1" 200 2698
[2020-10-20 14:56:33,996] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw?ran=0.0803395568252332 HTTP/1.1" 200 2993
[2020-10-20 14:56:40,278] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 14:56:40,293] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:56:40,305] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 40372
[2020-10-20 14:58:00,077] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:58:00,096] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 43746
[2020-10-20 14:58:00,152] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:00,158] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:00,159] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:00,161] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:58:00,489] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:00,489] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:00,501] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:00,504] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:00,505] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:00,506] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:58:23,219] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:58:23,240] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 43687
[2020-10-20 14:58:23,295] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:23,303] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:23,303] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:23,305] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:58:23,367] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:23,367] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:23,377] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:23,378] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:58:24,227] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:58:24,230] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 43687
[2020-10-20 14:58:24,262] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:24,269] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:24,271] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:24,272] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:58:24,306] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:58:24,307] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:58:24,313] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_res/pc/pf/pf.js
[2020-10-20 14:58:24,314] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_res/pc/pf/pf.js?ab_key=k9 HTTP/1.1" 404 2222
[2020-10-20 14:59:03,595] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:03,611] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 43563
[2020-10-20 14:59:03,707] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:59:03,707] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:59:04,483] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:04,487] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 43563
[2020-10-20 14:59:04,562] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:59:04,563] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:59:20,805] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:20,819] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42507
[2020-10-20 14:59:20,886] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:59:20,887] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:59:21,586] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:21,589] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42507
[2020-10-20 14:59:21,637] [base.py:93] [base:get_response] [WARNING]- Not Found: /ab_my/pc_pf.js
[2020-10-20 14:59:21,638] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /ab_my/pc_pf.js HTTP/1.1" 404 2200
[2020-10-20 14:59:30,403] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:30,416] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42423
[2020-10-20 14:59:37,334] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:37,350] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42343
[2020-10-20 14:59:44,070] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:44,098] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42246
[2020-10-20 14:59:44,804] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:44,807] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42246
[2020-10-20 14:59:45,505] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:45,509] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42246
[2020-10-20 14:59:45,771] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:45,775] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42246
[2020-10-20 14:59:46,540] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 14:59:46,543] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42246
[2020-10-20 15:00:05,062] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:00:05,077] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42149
[2020-10-20 15:00:06,131] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:00:06,134] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42149
[2020-10-20 15:00:07,029] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:00:07,032] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42149
[2020-10-20 15:00:07,355] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:00:07,358] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42149
[2020-10-20 15:00:07,565] [views.py:232] [views:index] [INFO]- 缓存中的key为：5fba0c7612a111ebb7f25800e3fb0696----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:00:07,569] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42149
[2020-10-20 15:13:52,317] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:5fba0c7612a111ebb7f25800e3fb0696
[2020-10-20 15:13:52,324] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 15:13:52,335] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 15:13:52,376] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2587
[2020-10-20 15:14:04,481] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 15:14:04,496] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:14:04,509] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42148
[2020-10-20 15:17:44,970] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:17:44,984] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42672
[2020-10-20 15:17:46,135] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:17:46,139] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42672
[2020-10-20 15:17:46,945] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:17:46,948] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42672
[2020-10-20 15:17:47,226] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:17:47,229] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42672
[2020-10-20 15:18:18,000] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:18:18,017] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42909
[2020-10-20 15:18:18,894] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:18:18,897] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42909
[2020-10-20 15:18:19,522] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:18:19,525] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42909
[2020-10-20 15:18:19,754] [views.py:232] [views:index] [INFO]- 缓存中的key为：cecb879c12a311eb85325800e3fb0696----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:18:19,758] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42909
[2020-10-20 15:21:03,651] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:cecb879c12a311eb85325800e3fb0696
[2020-10-20 15:21:03,659] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 15:21:03,669] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 15:21:03,706] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2610
[2020-10-20 15:21:15,429] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 15:21:15,444] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:21:15,456] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42774
[2020-10-20 15:22:18,305] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:22:18,316] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42760
[2020-10-20 15:22:19,487] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:22:19,489] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42760
[2020-10-20 15:22:20,112] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:22:20,114] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42760
[2020-10-20 15:22:20,353] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:22:20,356] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42760
[2020-10-20 15:25:33,547] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:33,561] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:34,388] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:34,392] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:34,721] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:34,724] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:35,078] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:35,081] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:35,384] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:35,386] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:35,829] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:35,841] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:36,282] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:36,285] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:25:36,528] [views.py:232] [views:index] [INFO]- 缓存中的key为：cfe3df5412a411eb91ba5800e3fb0696----浏览器中的key为:cfe3df5412a411eb91ba5800e3fb0696
[2020-10-20 15:25:36,531] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 42499
[2020-10-20 15:32:23,261] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:32:23,399] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/login/?next=/admin/novels/novelstype/add/ HTTP/1.1" 200 1901
[2020-10-20 15:32:32,008] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/login/?next=/admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:32:32,073] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:32:32,136] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:32:36,067] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/ HTTP/1.1" 200 3304
[2020-10-20 15:32:36,100] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:32:37,904] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:32:37,930] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:32:47,930] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 200 5737
[2020-10-20 15:32:47,957] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:33:19,016] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 200 5754
[2020-10-20 15:33:19,046] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:33:25,613] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/novels/novelstype/add/
Traceback (most recent call last):
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\options.py", line 574, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\views\decorators\cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\options.py", line 1553, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "D:\Envs\Django2020\lib\site-packages\django\utils\decorators.py", line 62, in _wrapper
    return bound_func(*args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\utils\decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\utils\decorators.py", line 58, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\options.py", line 1450, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\options.py", line 1490, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "D:\Envs\Django2020\lib\site-packages\django\contrib\admin\options.py", line 1026, in save_model
    obj.save()
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\base.py", line 729, in save
    force_update=force_update, update_fields=update_fields)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\base.py", line 759, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\base.py", line 842, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\base.py", line 880, in _do_insert
    using=using, raw=raw)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\query.py", line 1125, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\sql\compiler.py", line 1279, in execute_sql
    for sql, params in self.as_sql():
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\sql\compiler.py", line 1232, in as_sql
    for obj in self.query.objs
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\sql\compiler.py", line 1232, in <listcomp>
    for obj in self.query.objs
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\sql\compiler.py", line 1231, in <listcomp>
    [self.prepare_value(field, self.pre_save_val(field, obj)) for field in fields]
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\sql\compiler.py", line 1181, in pre_save_val
    return field.pre_save(obj, add=True)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\fields\files.py", line 287, in pre_save
    file.save(file.name, file.file, save=False)
  File "D:\Envs\Django2020\lib\site-packages\django\db\models\fields\files.py", line 87, in save
    self.name = self.storage.save(name, content, max_length=self.field.max_length)
  File "D:\Envs\Django2020\lib\site-packages\django\core\files\storage.py", line 48, in save
    name = self.get_available_name(name, max_length=max_length)
  File "D:\Envs\Django2020\lib\site-packages\django\core\files\storage.py", line 72, in get_available_name
    while self.exists(name) or (max_length and len(name) > max_length):
  File "D:\Envs\Django2020\lib\site-packages\django\core\files\storage.py", line 308, in exists
    return os.path.exists(self.path(name))
  File "D:\Envs\Django2020\lib\site-packages\django\core\files\storage.py", line 321, in path
    return safe_join(self.location, name)
  File "D:\Envs\Django2020\lib\site-packages\django\utils\functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "D:\Envs\Django2020\lib\site-packages\django\core\files\storage.py", line 201, in location
    return os.path.abspath(self.base_location)
  File "c:\programdata\anaconda3\lib\ntpath.py", line 543, in abspath
    path = os.fspath(path)
TypeError: expected str, bytes or os.PathLike object, not list
[2020-10-20 15:33:25,863] [basehttp.py:124] [basehttp:log_message] [ERROR]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 500 214267
[2020-10-20 15:35:29,421] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/ HTTP/1.1" 200 4683
[2020-10-20 15:35:31,776] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/ HTTP/1.1" 200 3304
[2020-10-20 15:35:31,814] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:35:33,442] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:35:33,476] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:35:51,186] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:35:51,245] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:35:51,278] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:35:51,324] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
[2020-10-20 15:37:23,983] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:37:24,043] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:37:24,079] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:37:47,607] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:37:47,661] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:37:47,693] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:48:53,080] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:48:53,132] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:48:53,167] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:49:53,716] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:49:53,772] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:49:53,812] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:50:36,812] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novelstype/add/ HTTP/1.1" 302 0
[2020-10-20 15:50:36,863] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5817
[2020-10-20 15:50:36,893] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:50:39,675] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:50:41,927] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:50:42,658] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/add/ HTTP/1.1" 200 5579
[2020-10-20 15:50:44,656] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/ HTTP/1.1" 200 3259
[2020-10-20 15:50:47,447] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novelstype/ HTTP/1.1" 200 5588
[2020-10-20 15:50:47,492] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:51:05,435] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/ HTTP/1.1" 200 3259
[2020-10-20 15:51:07,049] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/vicenovels/ HTTP/1.1" 200 3289
[2020-10-20 15:51:07,076] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:51:08,582] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/vicenovels/add/ HTTP/1.1" 200 6148
[2020-10-20 15:51:08,614] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:51:13,718] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/vicenovels/ HTTP/1.1" 200 3289
[2020-10-20 15:51:15,737] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/ HTTP/1.1" 200 3259
[2020-10-20 15:51:17,795] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novels/ HTTP/1.1" 200 3266
[2020-10-20 15:51:17,822] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:51:19,450] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novels/add/ HTTP/1.1" 200 8829
[2020-10-20 15:51:19,475] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:54:38,965] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /admin/novels/novels/add/ HTTP/1.1" 302 0
[2020-10-20 15:54:39,044] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/novels/ HTTP/1.1" 200 4611
[2020-10-20 15:54:39,084] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:54:42,425] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/ HTTP/1.1" 200 3259
[2020-10-20 15:54:44,752] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/vicenovels/ HTTP/1.1" 200 3289
[2020-10-20 15:54:44,799] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:54:46,298] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/vicenovels/add/ HTTP/1.1" 200 6196
[2020-10-20 15:54:46,331] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/jsi18n/ HTTP/1.1" 200 7678
[2020-10-20 15:54:55,494] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /admin/novels/ HTTP/1.1" 200 3259
[2020-10-20 16:01:18,963] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 16:01:19,008] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2247
[2020-10-20 16:01:31,799] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 16:01:31,813] [views.py:232] [views:index] [INFO]- 缓存中的key为：6f84656412aa11eb9be05800e3fb0696----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:01:31,817] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 9359
[2020-10-20 16:03:12,695] [views.py:232] [views:index] [INFO]- 缓存中的key为：6f84656412aa11eb9be05800e3fb0696----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:03:12,701] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /user/index
Traceback (most recent call last):
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\xinxi\PycharmProjects\Qinxiangshuge\user\views.py", line 238, in index
    return render(request, 'user/../templates/novel/index.html', {'user': user})
  File "D:\Envs\Django2020\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "D:\Envs\Django2020\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "D:\Envs\Django2020\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: user/../templates/novel/index.html
[2020-10-20 16:03:12,850] [basehttp.py:124] [basehttp:log_message] [ERROR]- "GET /user/index HTTP/1.1" 500 89935
[2020-10-20 16:03:33,385] [views.py:232] [views:index] [INFO]- 缓存中的key为：6f84656412aa11eb9be05800e3fb0696----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:03:33,401] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 9359
[2020-10-20 16:03:35,720] [views.py:232] [views:index] [INFO]- 缓存中的key为：6f84656412aa11eb9be05800e3fb0696----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:03:35,723] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 9359
[2020-10-20 16:05:19,915] [views.py:232] [views:index] [INFO]- 缓存中的key为：6f84656412aa11eb9be05800e3fb0696----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:05:19,930] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 9359
[2020-10-20 16:31:28,287] [views.py:232] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:6f84656412aa11eb9be05800e3fb0696
[2020-10-20 16:31:28,305] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 302 0
[2020-10-20 16:31:28,322] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 16:31:28,387] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2526
[2020-10-20 16:31:40,702] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 16:31:40,716] [views.py:232] [views:index] [INFO]- 缓存中的key为：a5fad10612ae11ebb40f5800e3fb0696----浏览器中的key为:a5fad10612ae11ebb40f5800e3fb0696
[2020-10-20 16:31:40,726] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/index HTTP/1.1" 200 9507
[2020-10-20 17:36:42,573] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:36:48,595] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:36:48,637] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2521
[2020-10-20 17:37:01,113] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /user/login
Traceback (most recent call last):
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\xinxi\PycharmProjects\Qinxiangshuge\user\views.py", line 45, in user_login
    response = redirect(reverse('user:index'))
  File "D:\Envs\Django2020\lib\site-packages\django\urls\base.py", line 88, in reverse
    return iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs))
  File "D:\Envs\Django2020\lib\site-packages\django\urls\resolvers.py", line 632, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'index' not found. 'index' is not a valid view function or pattern name.
[2020-10-20 17:37:01,242] [basehttp.py:124] [basehttp:log_message] [ERROR]- "POST /user/login HTTP/1.1" 500 88271
[2020-10-20 17:37:40,902] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:37:40,943] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2535
[2020-10-20 17:37:49,928] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 17:37:49,943] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:37:49,946] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:39:09,848] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:39:09,862] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:39:11,856] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:39:11,859] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:39:43,215] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:39:43,228] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:40:55,312] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:40:55,316] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:40:56,132] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:40:56,135] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:40:56,343] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:40:56,347] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:40:56,555] [views.py:22] [views:index] [INFO]- 缓存中的key为：e5d262e612b711eb81d35800e3fb0696----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:40:56,561] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9501
[2020-10-20 17:45:13,630] [views.py:22] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:e5d262e612b711eb81d35800e3fb0696
[2020-10-20 17:45:13,642] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 302 0
[2020-10-20 17:45:13,655] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:45:13,709] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2670
[2020-10-20 17:45:23,373] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 17:45:23,391] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:45:23,397] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 10113
[2020-10-20 17:46:02,280] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:46:02,286] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9789
[2020-10-20 17:46:03,404] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:46:03,410] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9789
[2020-10-20 17:46:47,481] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:46:47,488] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 10117
[2020-10-20 17:47:15,172] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:47:15,179] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:47:17,612] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:47:17,617] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:48:18,603] [views.py:22] [views:index] [INFO]- 缓存中的key为：f3af30e812b811eb9eda5800e3fb0696----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:48:18,609] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9979
[2020-10-20 17:50:58,832] [views.py:22] [views:index] [INFO]- 缓存中的key为：None----浏览器中的key为:f3af30e812b811eb9eda5800e3fb0696
[2020-10-20 17:50:58,841] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 302 0
[2020-10-20 17:50:58,850] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:50:58,920] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2743
[2020-10-20 17:51:08,076] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 17:51:08,091] [views.py:22] [views:index] [INFO]- 缓存中的key为：c16fae6412b911eb9a9b5800e3fb0696----浏览器中的key为:c16fae6412b911eb9a9b5800e3fb0696
[2020-10-20 17:51:08,097] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:52:42,454] [views.py:22] [views:index] [INFO]- 缓存中的key为：c16fae6412b911eb9a9b5800e3fb0696----浏览器中的key为:c16fae6412b911eb9a9b5800e3fb0696
[2020-10-20 17:52:42,471] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:53:47,583] [basehttp.py:124] [basehttp:log_message] [WARNING]- "GET /static/media/ HTTP/1.1" 404 1634
[2020-10-20 17:54:00,309] [views.py:22] [views:index] [INFO]- 缓存中的key为：c16fae6412b911eb9a9b5800e3fb0696----浏览器中的key为:c16fae6412b911eb9a9b5800e3fb0696
[2020-10-20 17:54:00,311] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /novels/index
Traceback (most recent call last):
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\xinxi\PycharmProjects\Qinxiangshuge\novels\views.py", line 32, in index
    print(Mtype.image)
NameError: name 'Mtype' is not defined
[2020-10-20 17:54:00,435] [basehttp.py:124] [basehttp:log_message] [ERROR]- "GET /novels/index HTTP/1.1" 500 75708
[2020-10-20 17:54:19,642] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /static/media/type/58950l.jpg HTTP/1.1" 200 335654
[2020-10-20 17:57:48,442] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/login HTTP/1.1" 200 4716
[2020-10-20 17:57:48,499] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /user/draw HTTP/1.1" 200 2837
[2020-10-20 17:57:57,262] [basehttp.py:124] [basehttp:log_message] [INFO]- "POST /user/login HTTP/1.1" 302 0
[2020-10-20 17:57:57,280] [views.py:23] [views:index] [INFO]- 缓存中的key为：b592437a12ba11ebb3d85800e3fb0696----浏览器中的key为:b592437a12ba11ebb3d85800e3fb0696
[2020-10-20 17:57:57,281] [exception.py:118] [exception:handle_uncaught_exception] [ERROR]- Internal Server Error: /novels/index
Traceback (most recent call last):
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\Envs\Django2020\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\xinxi\PycharmProjects\Qinxiangshuge\novels\views.py", line 33, in index
    print(MEDIA_URL+type.image)
TypeError: must be str, not ImageFieldFile
[2020-10-20 17:57:57,422] [basehttp.py:124] [basehttp:log_message] [ERROR]- "GET /novels/index HTTP/1.1" 500 75709
[2020-10-20 17:58:18,669] [views.py:23] [views:index] [INFO]- 缓存中的key为：b592437a12ba11ebb3d85800e3fb0696----浏览器中的key为:b592437a12ba11ebb3d85800e3fb0696
[2020-10-20 17:58:18,692] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:59:43,978] [views.py:23] [views:index] [INFO]- 缓存中的key为：b592437a12ba11ebb3d85800e3fb0696----浏览器中的key为:b592437a12ba11ebb3d85800e3fb0696
[2020-10-20 17:59:43,983] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:59:46,747] [views.py:23] [views:index] [INFO]- 缓存中的key为：b592437a12ba11ebb3d85800e3fb0696----浏览器中的key为:b592437a12ba11ebb3d85800e3fb0696
[2020-10-20 17:59:46,752] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
[2020-10-20 17:59:47,509] [views.py:23] [views:index] [INFO]- 缓存中的key为：b592437a12ba11ebb3d85800e3fb0696----浏览器中的key为:b592437a12ba11ebb3d85800e3fb0696
[2020-10-20 17:59:47,513] [basehttp.py:124] [basehttp:log_message] [INFO]- "GET /novels/index HTTP/1.1" 200 9919
